tests/ssl-opt.sh

```diff
@@ -1305,6 +1305,54 @@ requires_config_enabled MBEDTLS_SSL_MAX_FRAGMENT_LENGTH
             -C "found max_fragment_length extension"
 
 requires_config_enabled MBEDTLS_SSL_MAX_FRAGMENT_LENGTH
+run_test    "Max fragment length: enabled, default, larger message" \
+            "$P_SRV debug_level=3" \
+            "$P_CLI debug_level=3 request_size=20000" \
```

Comment:
This big of an overflow makes the test crash, I suggest to use a more subtle difference to simply fail instead of crashing. For example `request_size=16392` should be fine.

Reply:
@yanesca Could you elaborate on where this crashes? `ssl_client2` should deal with large request sizes gracefully, either handling them by allocating enough space, or rejecting immediately.

Reply:
@yanesca Thanks, done!